torch/itt.py

```diff
@@ -0,0 +1,39 @@
+try:
```

Comment:
This doesn't seem to warrant its own `torch.foo` namespace. Why don't we make these static methods on the context manager? E.g. something like:
```
def f():
  torch.profiler.ITT.mark("bbb")

with torch.profiler.ITT() as p:
  model(x)
  p.mark("aaa")
  f()
```
That way you could access it either from the active profiler or just from the module.

Reply:
Does this mean merging `emit_itt` into `torch.profiler.itt`?